checkov/sca_image/runner.py

```diff
@@ -183,6 +182,26 @@ def iterate_image_files(self, abs_fname: str, report: Report, runner_filter: Run
                                                          runner_filter=runner_filter)
                     merge_reports(report, image_report)
 
+    def get_report_from_scan_result(self, result: Dict[str, Any], dockerfile_path: str, rootless_file_path: str,
```

Comment:
also removing the report to be pasted in will make the need to change it again in my PR #3332, because I iterate over images and called `parse_vulns_to_records` function

Reply:
will fix it in my PR

Reply:
sorry =]